.github/workflows/deploy-arc-dashboard-prod.yaml

```diff
@@ -0,0 +1,16 @@
+name: deploy-arc-prod
+
+on:
+  workflow_dispatch
+
+jobs:
+  deploy:
+    permissions:
+      id-token: write
+      contents: read
+    uses: ./.github/workflows/build-and-deploy.yaml
+    with:
+      branch: main
```

Comment:
@syoung-smallwisdom / @jnelsonjava: do we plan to continue deploying ARC from the `main` branch (for production) and `feature` branch (for staging) or will we transition to using `production`/`staging` branches for ARC as well as Open Bridge?

Reply:
I think we'd want to transition to using the same pattern for all app-ids